packages/contracts-bedrock/contracts/L1/L1StandardBridge.sol

```diff
@@ -264,7 +264,7 @@ contract L1StandardBridge is StandardBridge, Semver {
         address _from,
         address _to,
         uint32 _minGasLimit,
-        bytes memory _extraData
+        bytes calldata _extraData
```

Comment:
This is how it's being done in the L2 bridge already:

https://github.com/ethereum-optimism/optimism/blob/3f07d1bc2a2a7b0339d839545de57f57702a582d/packages/contracts-bedrock/contracts/L2/L2StandardBridge.sol#L179-L185